frontend/src/components/page_content/profile/finances.js

```diff
@@ -103,37 +83,26 @@ class Finances extends React.Component {
           </Center>
           <hr />
           <Center>
-            <ToastButton tertiary label="Add Bill" />
+            <ToastButton tertiary label="Add Bills" />
           </Center>
           <hr />
           <Center>
-            <ToastButton tertiary label="Add Utility" />
+            <ToastButton tertiary label="Add Utilities" />
           </Center>
           <hr />
           <Center>
-            <ToastButton tertiary Button label="Add Insurance" />
+            <ToastButton tertiary Button label="Add Insurances" />
           </Center>
           <hr />
           <Center>
-            <ToastButton tertiary label="Add Loan / Debt" />
+            <ToastButton tertiary label="Add Loans / Debts" />
           </Center>
         </div>
 
         <div>
           <h4>Expenses</h4>
 
           <div className="row">
-            <div className="column">
```

Comment:
if you are deleting this, you should also delete the associated values in redux (state, reducer, actions, selectors) for shopping.description. (you can keep shopping, but description won't be used)